src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -603,7 +603,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
   renderWearablePreview = () => {
     const { type, contents } = this.state
     const isEmote = type === ItemType.EMOTE
-    const blob = contents ? toWearableWithBlobs({ contents, isEmote }) : undefined
+    const blob = contents ? toEmoteWithBlobs({ contents }) : undefined
```

Comment:
the method `renderWearablePreview` also computes the thumbnail for wearables. 
the blob should be something like this:
`const blob = contents ? (isEmote ? toEmoteWithBlobs({ contents }) : toWearableWithBlobs({ contents })) : undefined`

Reply:
good catch!